app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -367,14 +367,25 @@ class UserSession @Inject constructor(
         .map { if (it.isEmpty()) None else Just(it.first()) }
   }
 
+  @Deprecated(
+      message = "User can get logged out now and cleared; Don't use this function anymore",
+      replaceWith = ReplaceWith("loggedInUser")
+  )
   fun requireLoggedInUser(): Observable<User> {
-    return loggedInUser()
-        .map { (user) ->
-          if (user == null) {
-            throw AssertionError("User isn't logged in yet")
-          }
-          user
-        }
+    /*
```

Comment:
Should this make it to an existing/new ADR?

Reply:
We should; I'll add documentation task to Pivotal.

Reply:
https://www.pivotaltracker.com/story/show/167867802